pom.xml

```diff
@@ -53,23 +53,12 @@
                 <version>${apache.felix.scr.ds.annotations.version}</version>
             </dependency>
 
-            <dependency>
```

Comment:
Is there any reason to remove these dependencies

Reply:
These dependencies are not required in compile time.